### ISSUE TITLE
Error when pressing 'Run Image' in 'Docker Image' grid

### ISSUE THREAD

**Issue by votdev** [OpenMediaVault-Plugin-Developers/openmediavault-docker-gui#77]
After pressing the 'Run Image' the browser fires errors that a component 'hostip-1' can't be registered because of an already existing duplicate. The problems seems to be that the dialog excessively uses the component ``id`` field, but does not take care that this ID must be unique across the whole UI. It seems to me that the image i used (linuxserver/transmission) for testing has multiple port bindings which will break with the existing implementation of the dialog.

**Comment by ryecoaaron** [comment]
While I can't seem to duplicate the errors on the same version of Chromium, I do see what you are saying about the excessive ID use.

**Comment by ryecoaaron** [comment]
Looks like switching to itemId is the way to go?

**Comment by votdev** [comment]
Yes, i think so, but i did not know the code which is relevant. Note, the [itemId](https://docs.sencha.com/extjs/6.2.0/classic/Ext.Component.html#cfg-itemId) is only available within the container.
